Assets/Scripts/UI/ColorPalette.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Netherlands3D.Twin
+{
+    [Serializable]
+    public struct NamedColor
+    {
+        public string Name;
+        public Color Color;
+    }
+    
+    [CreateAssetMenu(menuName = "Netherlands3D/ColorPalette", fileName = "ColorPalette", order = 0)]
+    public class ColorPalette : ScriptableObject
+    {
+        public List<NamedColor> Colors = new();
```

Comment:
Jammer, ik had gehoopt dat elke color in een Scriptable object zou zijn zodat we die individuele kleuren ook in andere scripts kunnen gebruiken zonder dat we de hele Palette moeten inladen en op naam matchen

Reply:
dat hoeft niet, hij maakt nu dynamisch een pallet texture van de scriptable, waar dan een bepaalde greyscale waarde aan gekoppeld wordt. dit is gewoon een andere manier om de kleur op te vragen maar wordt niet direct gebruikt